config.yaml

```diff
@@ -139,39 +139,20 @@ analysis:
             params:
                 k: 100
 
-    downstream_analysis:
-        - name: GeneRecallCurve
-          params:
-            reference_genes_path: '/path/to/reference_genes.csv'
-            top_K: 300
-            plots_per_row: 3
-            features_selector:
-                name: ClasswiseAbs
-                params: {}
-        - name: Heatmap
-          params:
-            top_n_genes: 100
-        - name: RocAucCurve
-          params: {}
-        - name: DgePseudoBulk
-          params:
-              celltype_column: 'cell_type'
-              design_factor: 'disease'
-              factor_categories: ['Alzheimer disease', 'normal']
-              sum_column: 'donor_id'
-              cell_subsets: ['excitatory neuron', 'inhibitory interneuron', 'oligodendrocyte']
-              min_cell_threshold: 1
-              fold_change: 1.5
-              p_val: 0.05
-              save_plot: True
-        - name: DgeLMEM
-          params:
-             fixed_effect_column: 'disease'
-             fixed_effect_factors: ['Alzheimer disease', 'normal']
-             group: 'donor_id'
-             min_cell_threshold: 10
-             n_cpu: 4
-             gene_batch_size: 1000
-             coef_threshold: 0
-             p_val: 0.05
-             save_plot: True    
\ No newline at end of file
+    # downstream_analysis:
+    #     - name: GeneRecallCurve
+    #       params:
+    #         reference_genes_path: '/path/to/reference_genes.csv'
+    #         top_K: 300
+    #         plots_per_row: 3
+    #         features_selector:
```

Comment:
I think we should keep above params instead of kwargs, it will be easier for user to unuderstand params and modify.  In this case, little tedious to check whcih params are available and then write. We can keep the downstream_analysis commented above but with params atleast. Thoughts?

Reply:
But we shouldn't keep all params of all tasks in the main_config, this should only contain run related things. We already have separate docs and defaults set. Its like sklearn fit, it contains LOTS of parameters, but only shows few, if user wants to use more, he can check what params that class takes. 

Also the config gets over cluttered after adding, so I've added one, as a sample, and expect the users to understand how to use others

Reply:
okay lets remove the params for simplicity. User will refer config_README.md for params.